libs/build_metplus.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash                                                                                                                              
```

Comment:
This script does not adhere to the structure followed by the build scripts.
Please rewrite in a form that flows like the others.